docs/signals/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Snowplow Signals"
+sidebar_position: 8
+description: "An overview of Signals concepts."
+sidebar_label: "Signals"
+---
+
+Snowplow Signals empowers users to build more intelligent AI applications that can perceive and act on customer behavior in real-time. By aggregating engineer-defined attributes and facts, Signals enables Large Language Models (LLMs) to act intelligently and deliver personalized experiences.
+
+With real-time visibility into customer actions and seamless access to their complete history, Snowplow Signals simplifies the personalization of agentic applications, helping you build AI-driven solutions that understand and respond to your users.
+
+Built on top of Snowplow's behavioural data collection capabilities, integrating with Signals becomes straightforward once you grasp a few key concepts.
```

Comment:
behavioral, not behavioural

Reply:
do a search/replace

Reply:
Thanks - I'll rebase on the latest branch where we have a linter for US English spellings!